docs/RFCS/settings_table.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: Settings Table
+- Status: draft
+- Start Date: 2017-03-17
+- Authors: David Taylor
+- RFC PR:
+- Cockroach Issue:
+
+# Summary
+
+A system table of named settings with a caching accessor on each node to provide
+runtime-alterable execution parameters.
```

Comment:
There's an explicit discussion below of node-restart settings and flags, but basically those are explicitly not part of the "runtime-alterable" settings this is targeting (and probably make sense to stay as CLI flags).

w.r.t. scheduling changes: that feels to me like higher-level tooling, that sits on top of simpler primitives like this. Also, in the short term, you could use `cron`/`sleep`/etc to send a sql statement at a given time.